### PR TITLE
Added allowBlank for TV with URL type

### PR DESCRIPTION
**Pull request by Ruslan-Aleev**
### What does it do?
Added allowBlank for TV with URL type

url-req

### Why is it needed?
For some reason, allowBlank is not for all TVs, which is strange

### Related issue(s)/PR(s)
https://github.com/modxcms/revolution/issues/6521 - URL

### CODE REVIEW

**Review by Ibochkarev — APPROVED** [review]
Work!
image

File: manager/templates/default/element/tv/renders/input/url.tpl [hunk 1]
```diff
@@ -1,37 +1,48 @@
...
         ,listeners: { 'select': { fn:MODx.fireResourceFormChange, scope:this}}
     });
+    fld.wrap.applyStyles({
+        display: "inline-block"
+    });
 
-	fld.wrap.applyStyles({
-		display: "inline-block"
-	});
+    var fld = MODx.load({
```
Comment (Jako) [inline manager/templates/default/element/tv/renders/input/url.tpl:36]: Is this addition really needed?

Comment (Ruslan-Aleev) [reply manager/templates/default/element/tv/renders/input/url.tpl:36]: Well, this is a text field handler, it is for him that you need `allowBlank: false`, because links can be without the selected type in the drop-down list.
Or did I misunderstand you?

**Comment by garryn** [comment]
This seems a good addition @opengeek (subject to testing of course)

**Review by JoshuaLuckers — APPROVED** [review]
Works as intended.
